flink-table/flink-table-planner/src/test/java/org/apache/flink/table/planner/plan/nodes/exec/testutils/BatchRestoreTestBase.java

```diff
@@ -228,10 +228,9 @@ void loadAndRunCompiledPlan(TableTestProgram program, ExecNodeMetadata metadata)
 
         compiledPlan.execute().await();
         for (SinkTestStep sinkTestStep : program.getSetupSinkTestSteps()) {
-            List<String> expectedResults = getExpectedResults(sinkTestStep, sinkTestStep.name);
-            assertThat(expectedResults)
-                    .containsExactlyInAnyOrder(
-                            sinkTestStep.getExpectedAsStrings().toArray(new String[0]));
+            List<String> actualResults = getActualResults(sinkTestStep, sinkTestStep.name);
+            List<String> expectResults = sinkTestStep.getExpectedMaterializedResultsAsStrings();
+            assertThat(actualResults).containsExactlyInAnyOrderElementsOf(expectResults);
```

Comment:
@snuyanzin @twalthr just to draw attention to it, I switched this to `containsExactlyInAnyOrderElementsOf`.

While working on another test, I noticed that using `containsAll` just checks one direction of containment rather than list equality.

Reply:
Great that you spotted it early.